docs/services/sensors.md

```diff
@@ -0,0 +1,147 @@
+
+## API
+
+The Sensors Service supports the following methods:
+
+Method Name | Description
+----------- | -----------
+[`Sensors`](#sensors) | Returns a list of names of the available sensors.
+[`Readings`](#readings) | Returns a list of sensor readings from a given list of sensors.
+
+{{% alert title="Note" color="note" %}}
+
+The following code examples assume that you have a robot configured with a gripper, and that you add the required code to connect to your robot and import any required packages at the top of your code file.
+Go to your robot's **Code Sample** tab on the [Viam app](https://app.viam.com) for boilerplate code to connect to your robot.
+
+{{% /alert %}}
+
+### Sensors
+
+Returns a list of names of the available sensors.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` (Mapping[str, Any]) (*optional*): A generic struct, containing extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- `sensors` ([ResourceName](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.ResourceName)): Names of the available sensors of the robot.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/sensors/index.html#viam.services.sensors.SensorsClient.get_sensors).
+
+```python {class="line-numbers linkable-line-numbers"}
+# Access the Sensors Service
+sensors_svc = SensorsClient.from_robot(robot=robot, name="builtin")
+
+# Get available sensors
+sensors = sensors_svc.get_sensors()
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+- `extra` [(map\[string\]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [[resource.Name]](https://pkg.go.dev/go.viam.com/rdk/resource#Name): An array of sensor names.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/sensors#Service).
+
+```go {class="line-numbers linkable-line-numbers"}
+// Access the Sensors Service
+sensorsService, err := sensors.FromRobot(robot, "builtin")
+if err != nil {
+  logger.Fatal(err)
+}
+
+// Get available sensors
+sensor_names, err := sensorsService.Sensors(context.Background(), nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Readings
+
+Returns a list of sensor readings from a given list of sensors.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `sensors` ([ResourceName](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.ResourceName)): An array of sensor names for which to return readings.
+
+- `extra` (Mapping[str, Any]) (*optional*): A generic struct, containing extra options to pass to the underlying RPC call.
```

Comment:
I think this needs to be fixed and link put up here?

Reply:
What needs to be fixed and what link?